# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/DMABufSurface.cpp
# Commit: 4c4c96ad829d
# Full Hash: 4c4c96ad829d55b88ee52a320cab335ac2b86340
# Author: stransky <stransky@redhat.com>
# Date: 2025-02-05 16:56:59
# Regressor Bug: 1944490
# File Overlap Count: 1
# Description:
#   Bug 1944490 [Linux] Ensure DMABufFormats provides basic dmabuf formats r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D236688
# ==============================================================================

diff -r 25e327e7c9ad -r 4c4c96ad829d widget/gtk/DMABufSurface.cpp
--- a/widget/gtk/DMABufSurface.cpp	Tue Feb 04 20:14:58 2025 +0000
+++ b/widget/gtk/DMABufSurface.cpp	Tue Feb 04 20:44:48 2025 +0000
@@ -557,6 +557,8 @@
                                                      : GBM_FORMAT_XRGB8888;
   RefPtr<DRMFormat> format = GetDMABufDevice()->GetDRMFormat(mFOURCCFormat);
   if (!format) {
+    LOGDMABUF(("DMABufSurfaceRGBA::Create(): Missing drm format 0x%x!",
+               mFOURCCFormat));
     return false;
   }
   return Create(aWidth, aHeight, format, aDMABufSurfaceFlags);